transform/app/settings.py

```diff
@@ -108,8 +132,45 @@ def __init__(self, **kwargs):
                 "SEND_TO_SOLR or/and SEND_TO_S3 needs to be set to True. Otherwise, program will not have any effect"
             )
 
+        if self.CREATE_RELATIONS:
+            self.RELATIONS = self.get_relations_config()
+
         self.COLLECTIONS = self.get_collections_config()
 
+    def get_relations_config(self) -> dict:
+        """Get relations config"""
+
+        relations = {
+            self.TMP_DIRECTORY: {
+                self.DIRECTORY: "_tmp/",
+                self.COMBINED_PQ: "_tmp/combined_data.parquet",
+                self.ORGANIZATION_PQ: "_tmp/organization.parquet",
+                self.PROJECT_PQ: "_tmp/project.parquet",
+            },
+            self.DIRECTORIES_WITH_ADDITIONAL_COLUMNS: (self.ORGANISATION_PATH,),
+            self.SINGLE_DIRECTORIES: (
+                self.ORGANISATION_PATH,
+                self.PROJECT_PATH,
+            ),
+            self.DATA_DIRECTORIES: (
+                self.ORGANISATION_PATH,
+                self.PROJECT_PATH,
+                self.PUBLICATION_PATH,
+                self.DATASET_PATH,
+                self.SOFTWARE_PATH,
+                self.OTHER_RP_PATH,
+            ),
+            self.RESULT_RELATION_DIRECTORIES: (
+                self.RESULT_ORGANIZATION_PATH,
+                self.RESULT_PROJECT_PATH,
+            ),
+            self.ORGANIZATION_PROJECT_RELATION_DIRECTORIES: (
+                self.ORGANIZATION_PROJECT_PATH,
```

Comment:
that seems removable, doesnt do much

Reply:
In the function `load_relations`, located in `transform.app.transform.relations.process_data.py`, I pass a list or tuple of directories. Therefore, I have made it a tuple here. I mainly use this function with a list or tuple, and this is the only case where I use it with a single directory. If necessary, I can find a way to handle both cases without dividing them.